### ISSUE TITLE
Review Lab3

### ISSUE THREAD

**Issue by mattiaottoborgo** [hllqna/CI2024_lab3#2]
The choice of implementing A* using Manhattan distance is a clever start to solving this problem. I appreciate also the introduction of a variation of Manhattan distance to increase the speed of convergence. 
To further improve the algorithm, I would suggest to improve a different heuristic, even though it is not trivial to find. In general you have done a good work!
Small note for the future: results are understandable from the notebook, but I think it would be even nicer if you report them also in markdown format in a readme (this is more for future projects as well!)

Good Job!

**Comment by hllqna** [comment]
Indeed, finding another heuristic which would both be admissible and allow for more efficient solution of the puzzle was not an easy task.
I've added the results overview as suggested, thank you!
